google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriter.java

```diff
@@ -171,12 +177,6 @@ public boolean onError(BulkWriterException error) {
    */
   private boolean writesEnqueued = false;
```

Comment:
Since all accesses to writes enqueued are under lock, should we add a `@GuardedBy` annotation?

Reply:
I removed them since this now applies to all state. Based on the Android link, I added them back. There is also some tooling that catches violations when `GuardedBy` object is accessed under lock (i.e. in Android Studio's build), so it is likely always better to add this annotations.